### PR TITLE
Add DrWatson.jl example

### PR DESCRIPTION
**Pull request by ericneiva**
Hi, @fverdugo, @santiagobadia, this is the PR for the new tutorial on using `DrWatson.jl`. Please, review and merge whenever you have some time. Cheers!

### CODE REVIEW

File: deps/build.jl
Comment (fverdugo) [inline deps/build.jl:19]: I guess that this title will not fit into the left panel of the html page. Can you find a shorter one?

**Review by fverdugo — APPROVED** [review]
Thanks @ericneiva for the PR!

See my minor comment  (perhaps I have some further comments after merging and seeing the generated html pages)

File: deps/build.jl
Comment (amartinhuertas) [reply deps/build.jl:19]: I think that authomatise is without h, right?

**Comment by fverdugo** [comment]
@santiagobadia @amartinhuertas Do you find OK to add this to the tutorials? 

It is true that it does not cover new Gridap functionality, but it is useful anyway to Gridap users. 

For me it's acceptable to be merged.

File: deps/build.jl
Comment (ericneiva) [reply deps/build.jl]: > I guess that this title will not fit into the left panel of the html page. Can you find a shorter one?
Done.
> I think that authomatise is without h, right?
Correct.

**Comment by amartinhuertas** [comment]
> @santiagobadia @amartinhuertas Do you find OK to add this to the tutorials?

> It is true that it does not cover new Gridap functionality, but it is useful anyway to Gridap users.

> For me it's acceptable to be merged.

While it is true that it is not actually a tutorial that illustrates new Gridap functionality ... if we do not put it here, where do we put it in order to have visibility? Is there any DrWatson repo with examples where we can suggest to add ours?

**Comment by ericneiva** [comment]
> > @santiagobadia @amartinhuertas Do you find OK to add this to the tutorials?
> 
> > It is true that it does not cover new Gridap functionality, but it is useful anyway to Gridap users.
> 
> > For me it's acceptable to be merged.
> 
> While it is true that it is not actually a tutorial that illustrates new Gridap functionality ... if we do not put it here, where do we put it in order to have visibility? Is there any DrWatson repo with examples where we can suggest to add ours?

As far as I know there isn't an examples repo.

**Comment by fverdugo** [comment]
Hi @ericneiva, I think we are ready to merge after resolving the conflicts.

**Review by fverdugo — CHANGES_REQUESTED** [review]
Hi @ericneiva, I think we are ready to merge after resolving the conflicts.

**Comment by ericneiva** [comment]
Hi @fverdugo, conflicts resolved. Thanks for merging.

**Review by fverdugo — APPROVED** [review]

**Pull request merged**
